src/components/sections/action-item.tsx

```diff
@@ -55,40 +63,76 @@ export default function ActionItem({
   return (
     <TouchableOpacity
       onPress={() => onPress?.(!checked)}
-      style={[style.spaceBetween, topContainer]}
+      style={[
+        style.spaceBetween,
+        topContainer,
+        {
+          backgroundColor: interactiveColor
+            ? interactiveColor.background
+            : topContainer.backgroundColor,
+        },
+      ]}
       testID={testID}
       accessibilityRole={role}
       accessibilityState={{
         [stateName]: checked,
       }}
       {...accessibility}
     >
-      <ThemeText
-        type={
-          mode === 'heading-expand' ? 'body__primary--bold' : 'body__primary'
-        }
-        style={contentContainer}
-      >
-        {text}
-      </ThemeText>
-      <ActionModeIcon mode={mode} checked={checked} />
+      <View style={{flexShrink: 1}}>
+        <ThemeText
+          type={
+            mode === 'heading-expand' ? 'body__primary--bold' : 'body__primary'
+          }
+          style={[
+            contentContainer,
+            interactiveColor ? {color: interactiveColor.text} : undefined,
+          ]}
+        >
+          {text}
+        </ThemeText>
+        {subtext && (
+          <ThemeText
+            type="body__secondary"
+            color="secondary"
+            style={{marginTop: theme.spacings.medium}}
+          >
+            {subtext}
+          </ThemeText>
+        )}
+      </View>
+      <ActionModeIcon
+        mode={mode}
+        checked={checked}
+        color={interactiveColor ? color : undefined}
+      />
     </TouchableOpacity>
   );
 }
 
 function ActionModeIcon({
   mode,
   checked,
-}: Pick<ActionItemProps, 'mode' | 'checked'>) {
+  color,
+}: Pick<ActionItemProps, 'mode' | 'checked' | 'color'>) {
   const style = useHeaderExpandStyle();
   const {t} = useTranslation();
+  const {theme} = useTheme();
 
   switch (mode) {
     case 'check': {
       if (checked) {
-        return <ThemeIcon svg={Confirm} />;
+        return (
+          <ThemeIcon
+            svg={Confirm}
+            {...(color
```

Comment:
If you find the time you could make the parameter `colorType` accept a `ContrastColor`, as done on `ThemeText`. It is kinda technical debt that this hasn't been done yet.

Reply:
Seems like something that will lead to a lot small changes in lots of different corners of the app, so should probably be its own PR.